main/src/library/Fixpoint3/Ast/ExecutableRam.flix

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright 2025 Casper Dalgaard Nielsen
+ *                Adam Yasser Tallouzi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+///
+/// Purpose of `ExecutableRam`: Low level representation of the program to be executed.
+/// This representation is tightly coupled to the current interpreter e.g. `Ram`'s `NotBot`
+/// is removed since the interpreter handles it explicitly.
+///
+
+mod Fixpoint.Ast.ExecutableRam {
+    use Fixpoint3.Ast.Ram.{Predicates, RelSym, Search, IndexInformation}
+    use Fixpoint.Ast.Shared.{PredSym, BoxedDenotation => Denotation}
+    use Fixpoint3.Boxed;
+    use Fixpoint3.BoxingType.{UnifiedTypePos, RamIdToPos};
+    use Fixpoint.SubstitutePredSym
```

Comment:
Flix doesn't have support for `use` formatting, so make sure to sort these and remove any unused ones